### ISSUE TITLE
Unclear about pathing

### ISSUE THREAD

**Issue by m-w-andrews** [worldbank/Ethiopia-Corridors-IE#17]
I'm encountering consistent issues with the pathing in the master script and individual scripts in relation to the dropbox folder. 

For example:
'rawdata_file_path' assignment on line 21 of the master script leads to ~Data/RawData. This file path is then used in "01_create_points_globcover.R" to pull gadm boundary data with ~rawdata_file_path/GADM. However, the dropbox does not appear to have a GADM folder under ~Data/RawData but does have a GADM folder directly under ~Data. I'm not sure if this is the intended directory or not. 

~Data/RawData only seems to have soil moisture data so we may have to update the pathing code across the scripts. Thoughts? Let me know if I'm missing something.

Side-note: is this the right place to bring up something like this...?

**Comment by ramarty** [comment]
Yup, this is the exact right place to bring this up! Thanks for flagging! I recently reorganized dropbox without updating paths. Working on fixing -- will let you know once I've cleaned things up!

**Comment by ramarty** [comment]
@m-w-andrews Most of the paths should be fixed. Reorganized things a bit here https://github.com/worldbank/Ethiopia-Corridors-IE/tree/master/DataWork/Panel%20Data%20RSDP%20Impacts

There's two main folders:

- `01_extract_data_to_datasets` - Creates "base" dataset (e.g., grid of cells near roads), and extracts variables to that dataset (e.g., extracts nighttime lights). Works for both grid and woreda level. Creates a separate dataset for each dataset type... so grid level dataset with VIIRs extracted, NDVI extracted, etc.

- `02_prep_dataset_and_analysis` - In here, I created folders for grid/woreda & for long difference and panel. Working on cleaning stuff up here. But in `grid_panel`, for example, we (1) merge relevant datasets together, (2) clean up so have variables we need for grid-panel analysis and (3) have analysis code. My thinking is that we'll use different variables depending on whether it's grid or woreda, and whether it's panel or long difference.

(Note -- panel = all years; long difference = still a panel, but with just 2 time periods... eg., 1996 and 2016).
